README.md

```diff
@@ -1,11 +1,10 @@
-# distro-tools
-Cloud-native build and release tools tailored to building, releasing and maintaining Linux distributions and forks
+# Peridot
+Named after the Gemstone,  Peridot (pronounced  - PERR-ih-dot) is a cloud-native build and release tool used for building, releasing and maintaining Linux distributions and forks.
 
 ## Structure
 __Other components pending__
 
 * publisher - `Composer for Peridot (currently only includes legacy mode)`
-* peridot - `Modern build system`
```

Comment:
This should stay. Peridot is one of the tools in this repository. Ex. check secparse

Reply:
This is since this is a section detailing project structure. This will be expanded upon so worth keeping